api/test/unit/validation/attributeValidator.test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+var expect = require("chai").expect;
+var fs = require('fs');
+var rewire = require('rewire');
+var logger = require('../../../lib/log/logger');
+var attributeValidator = require('../../../lib/validation/attributeValidator');
+var attributeValidatorPrivate = rewire('../../../lib/validation/attributeValidator');
+
+describe('Validating Policy JSON properties',function(){
+    
+    var fakePolicy;
+    beforeEach(function(){
+      fakePolicy = JSON.parse(fs.readFileSync(__dirname+'/../fakePolicy.json', 'utf8'));
+    });
+    
+    it('Should validate the policy JSON successfully',function(){
+      attributeValidator.validatePolicy(fakePolicy,function(result){
+        expect(result).to.be.empty;
+      });
+    });
+    it('Should fail to validate the policy due to wrong Instance count',function(){
```

Comment:
That's what the 'expect check' says but the 'it' doesn't. Will fix in the next PR.